packages/tauri/build-tauri-conf.js

```diff
@@ -0,0 +1,40 @@
+const { resolve } = require("path");
+const { writeFileSync, readFileSync } = require("fs");
+const TOML = require("@iarna/toml");
+
+require("dotenv").config();
+
+const rootDir = resolve(__dirname, "../..");
+const assetsDir = resolve(rootDir, process.env.PL_ASSETS_DIR || "assets");
+
+const tauriConfigFilePath = resolve(__dirname, "src-tauri", "tauri.conf.json");
+const cargoTomlFilePath = resolve(__dirname, "src-tauri", "Cargo.toml");
+const packageFilePath = resolve(__dirname, "package.json");
+const manifestFilePath = resolve(assetsDir, "manifest.json");
+
+const tauriConfig = require(tauriConfigFilePath);
+const { version } = require(packageFilePath);
+const manifest = require(manifestFilePath);
+
+const vendorVersion = process.env.PL_VENDOR_VERSION || version;
+const vendorName = manifest.name;
+const vendorNameLowercase = vendorName.toLowerCase();
+const vendorBaseUrl = process.env.PL_VENDOR_BASE_URL || "https://github.com/padloc/padloc";
```

Comment:
This is a new variable that I haven't set in GitHub actions because it'll be the default in this repo.